### PR TITLE
Consolidate in EntityDetails to a single source of entity information

### PR DESCRIPTION
**Pull request by kadams54**

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3243/
Test PASSed.

**Comment by kadams54** [comment]
## QA

Make sure entity details still loads OK. Play around with loading multiple entities.

**Comment by hatched** [comment]
:+1: QA :ok:

File: jujugui/static/gui/src/app/components/entity-details/test-entity-details.js [hunk 1]
```diff
@@ -36,11 +36,14 @@ describe('EntityDetails', function() {
       promulgated: true,
       id: 'spinach',
       type: 'charm',
-      iconPath: 'icon.svg',
+      iconPath: 'data:image/gif;base64,',
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/entity-details/test-entity-details.js]: Nice!

**Comment by kadams54** [comment]
Thanks! :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**